app/views/shared/_order_detail_cell.html.haml

```diff
@@ -6,6 +6,10 @@
     - if show_reservation && od.reservation
       %br
       %em= od.reservation
+
+    - if od.product.offline?
```

Comment:
I'll take it out. I could see a method like `Reservation#overlaps_downtime?` (or something with a better name) being useful here if we want to revive the warning sign here.